exercises/dominoes/.meta/src/reference/java/Dominoes.java

```diff
@@ -0,0 +1,75 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+
+class Dominoes {
+	
+	public static void main(String[] args) {
```

Comment:
As described in our [policies doc](https://github.com/exercism/java/blob/master/POLICIES.md#prefer-instance-methods), we prefer exercise implementations to have instance methods. It would be great if you could change this to use instance methods to be consistent with the other exercises 🙂 

Also, why do you need a main method which prints to the console?

Reply:
The main method was just for testing. I'll remove that.

By instance methods you mean to make them not static, right?

Reply:
Yes, that's right 🙂